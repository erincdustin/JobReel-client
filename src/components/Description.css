.app-description{
  border: 1px black solid;
  width: 60%;
  margin: 25px auto;
}

.app-description h1{
  font-size: 26px;
}

.app-description h2{
  font-size: 20px;
}