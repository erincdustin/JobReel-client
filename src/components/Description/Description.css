@media (min-width: 761px) {
  .app-description {
    border: 1px black solid;
    padding: 5%;
    grid-column-start: 1;
    grid-column-end: 5;
    width: 320px;
    min-height: 350px;
    border-radius: 10px;
    background: #3b3a50;
    color: #eeebd9;
    box-shadow: 0.5em 0.5em 0.2em rgb(48, 39, 39);
  }
  
  .app-description > h1{
    font-size: 26px;
    text-align: left;
  }
  
  .app-description > h2{
    font-size: 20px;
    width: 80%;
    margin: auto;
    text-align: center;
  }
  
  .app-description > p {
    margin-top: 1.5em; 
    line-height: 150%;
  }
}

@media (min-width: 320px) and (max-width: 760px) {
  .app-description {
    border: 1px black solid;
    padding: 5%;
    grid-column-start: 1;
    grid-column-end: 9;
    width: 100%;
    min-height: 350px;
    border-radius: 10px;
    background: #3b3a50;
    color: #eeebd9;
    box-shadow: 0.5em 0.5em 0.2em rgb(48, 39, 39);
  }
  
  .app-description > h1{
    font-size: 26px;
    text-align: left;
  }
  
  .app-description > h2{
    font-size: 20px;
    width: 80%;
    margin: auto;
    text-align: center;
  }
  
  .app-description > p {
    margin-top: 1.5em; 
    line-height: 150%;
  }
}

/* @media (min-width: 481px) and (max-width: 960px) {
  .app-description {
    border: 1px black solid;
    padding: 5%;
    grid-column-start: 1;
    grid-column-end: 5;
    width: 320px;
    min-height: 350px;
    border-radius: 10px;
    background: #3b3a50;
    color: #eeebd9;
    box-shadow: 0.5em 0.5em 0.2em rgb(48, 39, 39);
  }
  
  .app-description > h1{
    font-size: 26px;
    text-align: left;
  }
  
  .app-description > h2{
    font-size: 20px;
    width: 80%;
    margin: auto;
    text-align: center;
  }
  
  .app-description > p {
    margin-top: 1.5em; 
    line-height: 150%;
  }
} */

