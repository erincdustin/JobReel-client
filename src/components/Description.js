import React from 'react';
import './Description.css'

export default function Description(props){
  return(
    <section className="app-description">
      <h1>JobReel</h1>
      <h2>description text</h2>
    </section>
  )
}