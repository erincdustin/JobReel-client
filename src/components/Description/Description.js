import React from 'react';
import './Description.css'

export default function Description(props){
  return(
    <div className='app-description'>
      <h1>JobReel</h1>
      <h2>Cast a wide net for your dream job.</h2>
      <p>There are a lot of moving pieces in a job hunt.  
        Keep yourself organized and on task with JobReel's 
        all-encompassing interface.  Search new postings and 
        expand your network with our integrated job and 
        industry event search, and track your application 
        process from start to finish through the dashboard.  
      </p>
    </div>
  )
}