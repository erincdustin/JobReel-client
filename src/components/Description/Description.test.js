import React from 'react'
import ReactDOM from 'react-dom'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import Description from './Description'

describe('Description smoke test', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div')
    ReactDOM.render(<Description />, div)
    ReactDOM.unmountComponentAtNode(div)
  })
})

describe('Description snapshot test', () => {
  it('render the UI as expected', () => {
    const wrapper = shallow(<Description />)
    expect(toJson(wrapper)).toMatchSnapshot()
  })
})