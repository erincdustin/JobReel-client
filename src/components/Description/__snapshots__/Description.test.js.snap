// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Description snapshot test render the UI as expected 1`] = `
<section
  className="app-description"
>
  <h1>
    JobReel
  </h1>
  <h2>
    Cast a wide net for your dream job.
  </h2>
  <p>
    There are a lot of moving pieces in a job hunt. Keep yourself organized and on task with JobReel's all-encompassing interface.  Search new postings and expand your network with our integrated job and industry event search, and track your application process from start to finish through the dashboard.
  </p>
</section>
`;
